ompi/mpi/c/init_thread.c

```diff
@@ -56,7 +57,13 @@ int MPI_Init_thread(int *argc, char ***argv, int required,
      */
     if( (MPI_THREAD_SINGLE == required) || (MPI_THREAD_SERIALIZED == required) ||
         (MPI_THREAD_FUNNELED == required) || (MPI_THREAD_MULTIPLE == required) ) {
-        safe_required = required;
+
```

Comment:
Funny that we support OMPI_MPI_THREAD_LEVEL in MPI_Init but not in MPI_Init_thread.